packages/contracts/contracts/TransactionManager.sol

```diff
@@ -500,31 +484,22 @@ contract TransactionManager is ReentrancyGuard, ITransactionManager {
         require(msg.sender == txData.router, "cancel: ROUTER_MUST_CANCEL");
 
         // Return totality of locked funds to provided fallbacl
-        require(
-          LibAsset.transferAsset(txData.sendingAssetId, payable(txData.sendingChainFallback), txData.amount),
-          "cancel: TRANSFER_FAILED"
-        );
+        LibAsset.transferAsset(txData.sendingAssetId, payable(txData.sendingChainFallback), txData.amount);
       } else {
         // When the user could be unlocking funds through a relayer, validate
         // their signature and payout the relayer.
         if (relayerFee > 0) {
           require(recoverCancelSignature(txData, relayerFee, signature) == txData.user, "cancel: INVALID_SIGNATURE");
 
-          require(
-            LibAsset.transferAsset(txData.receivingAssetId, payable(msg.sender), relayerFee),
-            "cancel: FEE_TRANSFER_FAILED"
-          );
+          LibAsset.transferAsset(txData.receivingAssetId, payable(msg.sender), relayerFee);
```

Comment:
I know we are fixing the issues, mentioned on PR. But just wanted to ask if we should fix the typo here?
receivingAssetId to sendingAssetId

Reply:
nvm checked out the #25 pr.